src/client/viewSandbox.js

```diff
@@ -456,7 +458,17 @@ const init = () => {
 
   // There are some timing issues between the CoralUI panelstack and CodeMirror rendering.
   // Without this, sometimes the CodeMirror editors don't render correctly.
-  document.querySelector('coral-panelstack').addEventListener('coral-panelstack:change', () => {
+  document.querySelector('.coral-TabList').addEventListener('click', (e) => {
+    if (e.target.className === 'coral-TabList') {
+      return;
+    }
+    document.querySelector('.coral-Tab.is-selected').classList.remove('is-selected');
+    document.querySelector('.coral-Panel.is-selected').classList.remove('is-selected');
+
+    document.getElementById(e.target.id).classList.add('is-selected');
+    document.getElementById(e.target.id.replace('Tab', 'Panel'))
+      .classList.add('is-selected');
```

Comment:
ID string manipulation is pretty brittle. I would prefer we put a `data-panel-id` attribute on the the tabs and have the values be the IDs of their respective panels. Alternatively, we could use the index of the tab to retrieve the panel that's at the same index.